go.mod

```diff
@@ -5,6 +5,5 @@ go 1.15
 require (
 	github.com/gocarina/gocsv v0.0.0-20201103164230-b291445e0dd2
 	github.com/rinor/jorcli v0.0.0-20201117192102-2a69360d3a83
```

Comment:
I guess we'll get to this one in time, too.

Reply:
Aaaagh, why it doesn't update itself?? 😞

Reply:
Yeah, we can integrate it in the same way at some point when needed.